frontend/public/locales/en/gcweb.json

```
@@ -122,5 +122,8 @@
       "label": "Year",
       "placeholder": "YYYY"
     }
+  },
+  "screen-reader": {
+    "newtab": "Opens in a new tab"
```

Should be in lowercase.

```suggestion
    "newtab": "opens in a new tab"
```